b2sdk/account_info/abstract.py

```diff
@@ -99,6 +100,23 @@ def clear_bucket_upload_data(self, bucket_id):
         :param str bucket_id: a bucket ID
         """
 
+    def check_current_credentials(self, key_id, key, realm):
+        """
+        Check whether provided key_id, key and realm are the credentials cached at the moment.
+
+        :param str key_id: application key ID used to authenticate
+        :param str key: application key
+        :param str realm: authorization realm
+        :rtype: bool
+        """
+        try:
+            return (
+                self.get_application_key() == key and self.get_application_key_id() == key_id and
```

Comment:
do we really need to check the key secret? I think checking `key_id` and `realm` should be enough

Reply:
Yeah, since it's not possible to have the same `key_id` and different `keys`, this brings nothing to the table, except for breaking on some weird configuration errors. So Ockham says comparing the `key` is bad.